app/resources/v1/pacbio/run_resource.rb

```diff
@@ -7,41 +7,37 @@ class RunResource < JSONAPI::Resource
       model_name 'Pacbio::Run'
 
       attributes :name, :sequencing_kit_box_barcode, :dna_control_complex_box_barcode,
-                 :system_name, :created_at, :state, :comments, :all_wells_have_pools,
-                 :pacbio_smrt_link_version_id
-
+                 :system_name, :created_at, :state, :comments,
+                 :pacbio_smrt_link_version_id, :well_attributes
       has_one :plate, foreign_key_on: :related, foreign_key: 'pacbio_run_id',
                       class_name: 'Runs::Plate'
 
       has_one :smrt_link_version, foreign_key: 'pacbio_smrt_link_version_id'
 
       filters :name, :state
+
       paginator :paged
 
+      PERMITTED_WELL_PARAMETERS = %i[id
+                                     row column ccs_analysis_output generate_hifi
+                                     ccs_analysis_output_include_low_quality_reads
+                                     include_fivemc_calls_in_cpg_motifs comment
+                                     ccs_analysis_output_include_kinetics_information
+                                     demultiplex_barcodes on_plate_loading_concentration
+                                     binding_kit_box_barcode pre_extension_time
+                                     loading_target_p1_plus_p2 movie_time].freeze
+
+      after_create :publish_messages
```

Comment:
can `after_save` replace `after_create` and `after_update`

Reply:
Cheers